packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -168,6 +163,16 @@ export function useGridEditRows(apiRef: GridApiRef) {
     [apiRef, forceUpdate, logger, setGridState],
   );
 
+  const handleCellEditPropsChange = React.useCallback(
+    (params: GridEditCellPropsParams, event?: React.SyntheticEvent) => {
+      if (event?.isPropagationStopped()) {
```

Comment:
Is that not handled in the publishEvent?

Reply:
No, `publishEvent` prevents from publishing an event if the given event object has the propagation stopped. Once the event is published, it will call all the listeners. If the user wants to stop the propagation of an event in `onEditCellChange`, we need to check by hand if the propagation was stopped.

Reply:
Yes so one way to do it, is to override the emit behavior of the event emitter so we avoid this check everywhere 🤔